aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -0,0 +1,193 @@
+/// Provides the configuration for staking and rewards
+module aptos_framework::staking_config {
+    use std::error;
+    use aptos_framework::system_addresses;
+
+    /// Invalid required stake lockup value.
+    const EINVALID_LOCKUP_VALUE: u64 = 1;
+    /// Invalid rewards rate.
+    const EINVALID_REWARDS_RATE: u64 = 2;
+    /// Invalid required stake range, usually happens if min > max.
+    const EINVALID_STAKE_RANGE: u64 = 3;
+
+    /// Validator set configurations that will be stored with the @aptos_framework account.
+    struct StakingConfig has copy, drop, key {
```

Comment:
Does this need copy and drop because get() returns StakingConfig?

Reply:
yes